source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,101 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to sync data between a replica set and a 
```

Comment:
Since this is the first mention of the word "sync", I would use the full term (and then use sync for the rest one the page like you already have!)

```suggestion
This page outlines the procedure to synchronize data between a replica set and a 
```

Reply:
Good idea, changing now!